package.json

```diff
@@ -2,9 +2,14 @@
   "name": "drf-js",
   "version": "0.0.0",
   "description": "A Django Rest Framework js wrapper library",
-  "main": "index.js",
+  "main": "drf.js",
   "scripts": {
-    "test": "echo test"
+    "test": "npm run eslint; npm run jscs; npm run karma-single",
```

Comment:
I didn't think semicolons were allowed in these scripts, at least npm erred for me